### DISCUSSION TITLE
Mailto link support for URL columns in Table fields

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8996] [Ideas]
> Awesome! Any chance we could also get this ability when the URL field is within a Table field? Thanks

_Originally posted by @melissadoyon in https://github.com/craftcms/cms/issues/5497#issuecomment-773449804_

**Comment by jamiematrix** [comment]
Can I add Telephone to the list too? Setting the URL field type to Telephone gives the error "fieldname is not a valid URL" when saving a phone number.
